spark.md

```diff
@@ -29,6 +29,11 @@ Approaches:
 - https://github.com/mozilla/jupyter-spark
 - [Progress monitor](https://github.com/jupyter/jupyter_client/issues/209#issuecomment-260789525)
 
+Progress:
+
+* [ProgressBar](https://github.com/ipython/ipython/blob/345d51130aec9644dbeb08c3e54d93f83babbf3f/IPython/core/display.py#L732) is now a builtin on the IPython library
```

Comment:
How about "is now available in the IPython library"?

Reply:
Good point, it's definitely not a builtin. Silly me.